app-web/src/store/reducers/siphon.js

```diff
@@ -29,6 +29,28 @@ const initialState = {
 
 const mapWithCallback = (array, cb) => array.map(cb);
 
+/**
+ * from an array of siphon positions
+ * [collection, source, resource]
+ * we return an integer by weighting each index to a power of ten
+ * weighting decreases left to right
+ * @param {Array} position
+ * @returns {String} an address of the position based on weight
+ * [0, 2, 5] = 100.30.6
+ */
+export const getTruePositionFromWeightedScale = position => {
```

Comment:
Often my linter tells me to not put `{ }` around a return statement but rater delete them and the word `return`.

Reply:
great point, thanks. I've changed that.